satpy/tests/reader_tests/test_vii_l1b_nc.py

```diff
@@ -151,13 +155,13 @@ def test_functions(self):
         calibrated_variable = self.reader._perform_calibration(variable,
                                                                {'calibration': 'brightness_temperature',
                                                                 'chan_thermal_index': 3})
-        expected_values = np.ones((72, 600)) * 302007.42728603
+        expected_values = np.ones((600, 72)) * 1101.10413712
```

Comment:
Just for my information, what is the unit here? 1101 is not K right?

Reply:
The result would technically be in K but these are dummy values and have no physical meaning.

Reply:
You can write this as `np.full((600, 72), 1101.10413712)`.

Reply:
Thanks - I will update accordingly.